views/private_subnet/show.erb

```diff
@@ -57,13 +57,15 @@
           <tr>
             <td class="whitespace-nowrap py-4 pl-4 pr-3 text-sm font-medium text-gray-900 sm:pl-6" scope="row"><%= nic[:name] %></td>
             <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
-              <span class="copy-content cursor-pointer" data-content="<%= nic[:private_ipv4] %>" data-message="Copied Private IPv4">
+              <span class="copy-content" data-content="<%= nic[:private_ipv4] %>" data-message="Copied Private IPv4">
                 <%= nic[:private_ipv4] %>
+                <img src="/icons/hero-clipboard-document.svg" width="20px" class="inline-block copy-button cursor-pointer"/>
               </span>
             </td>
             <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
-              <span class="copy-content cursor-pointer" data-content="<%= nic[:private_ipv6] %>" data-message="Copied Private IPv6">
+              <span class="copy-content" data-content="<%= nic[:private_ipv6] %>" data-message="Copied Private IPv6">
                 <%= nic[:private_ipv6] %>
+                <img src="/icons/hero-clipboard-document.svg" width="20px" class="inline-block copy-button cursor-pointer"/>
```

Comment:
You can move the `copy-content` span to an `erb` component. It may have `content`, `message`, and `body` parameters.